index.md

```diff
@@ -69,3 +71,5 @@ _**Legend:** A = Accepted | D = Draft | R = Review | V = Voting | X = Rejected_
 [psr9]: https://github.com/php-fig/fig-standards/blob/master/proposed/security-disclosure-publication.md
 [psr10]: https://github.com/php-fig/fig-standards/pull/473
 [psr11]: https://github.com/container-interop/fig-standards/blob/master/proposed/container.md
+[psr11]: https://github.com/container-interop/fig-standards/blob/master/proposed/container.md
+[psr12]: https://github.com/cs-extended/fig-standards/blob/pre-draft/proposed/extended-coding-style-guide.md
```

Comment:
This will need updating to https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md once #611 is merged (Which should be now as it's now passed)